.github/workflows/afd-domain-renewal.yml

```diff
@@ -0,0 +1,94 @@
+name: AFD Domain Certificate Renewal
+
+on:
+  workflow_dispatch:
+    inputs:
+      resource_group:
```

Comment:
let's remove this option from the workflow. While you can use it if manually running the script, the workflow is just to check all the prod front doors